tests/test_F0_output.py

```diff
@@ -101,7 +102,7 @@ def test_store_barchart_for_capacities_no_additional_capacities(self):
                 )
             },
         }
-        dict_scalar_capacities.update(DICT_PLOTS)
+        dict_scalar_capacities.update(copy.deepcopy(DICT_PLOTS))
```

Comment:
Oh yeah, I wondered when we might run into copy/deepcopy issues with our dict based structure